x-pack/plugins/synthetics/common/runtime_types/monitor_management/monitor_types.ts

```diff
@@ -229,7 +231,7 @@ export const BrowserSensitiveSimpleFieldsCodec = t.intersection([
     [ConfigKey.SOURCE_ZIP_USERNAME]: t.string,
     [ConfigKey.SOURCE_ZIP_PASSWORD]: t.string,
     [ConfigKey.PARAMS]: t.string,
-    [ConfigKey.URLS]: t.union([t.string, t.null]),
+    [ConfigKey.URLS]: t.union([t.string, t.undefined]),
```

Comment:
Allowing for undefined values in encrypted saved objects can cause unintended consequences with decryption from the disruption of AAD. Why are we changing this back to undefined? The change to null fixed a previously found bug.